django_socio_grpc/services.py

```diff
@@ -22,6 +22,8 @@ def __init__(self, **kwargs):
             setattr(self, key, value)
 
     def perform_authentication(self):
+        if len(self.authentication_classes) == 0:
```

Comment:
@MrDiben to make the test work I did that. I let you see the best practices for that